vnext/Shared/Modules/BlobModule.cpp

```diff
@@ -0,0 +1,222 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#include "BlobModule.h"
+
+#include <Modules/IWebSocketModuleProxy.h>
+#include <unicode.h>
+
+// React Native
+#include <cxxreact/Instance.h>
+#include <cxxreact/JsArgumentHelpers.h>
+
+// Windows API
+#include <winrt/Windows.Foundation.h>
+#include <winrt/Windows.Security.Cryptography.h>
+
+using namespace facebook::xplat;
+
+using facebook::react::Instance;
+using folly::dynamic;
+using std::lock_guard;
+using std::mutex;
+using std::string;
+using std::vector;
+using std::weak_ptr;
+using winrt::Windows::Foundation::GuidHelper;
+using winrt::Windows::Security::Cryptography::CryptographicBuffer;
+
+namespace {
+constexpr char moduleName[] = "BlobModule";
+constexpr char blobURIScheme[] = "blob";
+
+weak_ptr<Microsoft::React::IWebSocketModuleContentHandler> s_contentHandler;
+} // namespace
+
+namespace Microsoft::React {
+
+#pragma region BlobModule
+
+BlobModule::BlobModule() noexcept : m_sharedState{std::make_shared<SharedState>()} {
+  m_contentHandler = std::static_pointer_cast<BlobWebSocketModuleContentHandler>(s_contentHandler.lock());
+  if (!m_contentHandler) {
+    m_contentHandler = std::make_shared<BlobWebSocketModuleContentHandler>();
+    s_contentHandler = m_contentHandler;
+  }
```

Comment:
This code is not thread safe. In order to access any static variable from different threads we must have a static mutex. 
Why do we have multiple threads? 
There can be multiple RN instances and each will have its own thread. The s_contentHandler is going to be shared across multiple instances.